numpy/testing/tests/test_utils.py

```diff
@@ -341,24 +341,6 @@ def test_complex(self):
         self._assert_func(x, x)
         self._test_not_equal(x, y)
 
-    def test_error_message(self):
```

Comment:
Seems silly to have the duplicate test, and the above test says "moved here" not "copied here".
"# 2018-04-29: moved here from testing.tests.test_utils." 

I'll investigate history if I have time